src/components/frame/Extensions/pages/AdvancedPage/AdvancedPage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { useExtensionsData } from '~hooks';
+import { SpinnerLoader } from '~shared/Preloaders';
+
+const displayName = 'frame.Extensions.pages.AdvancedPage';
+
+const AdvancedPage: FC = () => {
+  const { loading } = useExtensionsData();
```

Comment:
please create separate component for displaying spinner and import it in all places